### DISCUSSION TITLE
Protect android app bundle (AAB)

### DISCUSSION THREAD

**Discussion by redmare-27** [REAndroid/APKEditor#81] [Ideas]
Hi APKEditor Team,

APKEditor is one of the best recompiler tools that I have used. 
The protect feature is very useful to protect APK resources against almost all known decompile/modify tools.
But it doesn't support AAB protection. If we distribute an app to Google Play, we use the AAB format, right? 
Can you develop AAB protection for the next feature?

Thank you

**Comment by REAndroid** [comment]
I don't think google will like it, but if you would like to experiment we can try something
